### DISCUSSION TITLE
Special Cartridge Support for Game Boy Family

### DISCUSSION THREAD

**Discussion by PW5190** [sanni/cartreader#511] [Ideas & Suggestions]
I have several questions about the Cartridge Reader. Can it dump cartridges made for the Game Boy family of systems (GB, GBC and GBA) with special features built-in, such as Rumble/Vibration (i.e. _Pokémon Pinball_, _Drill Dozer_), Solar (_Boktai series_), Motion/Tilt/Gyro (i.e. _Kirby Tilt 'n' Tumble_, _WarioWare: Twisted!_), Pocket Sonar (by _Bandai_), _e-Reader_, etc.? Also, can it dump the _Game Boy Advance Video_ library of cartridges (you know, TV without the TV?), as well as the ones with Matrix Memory chips (especially the ones found in DreamWorks movies and certain late-release TV carts)? If the Matrix Memory cartridges couldn't be dumped this way with the current firmware, would that feature be implemented in another firmware update?

**Comment by PsyK0p4T** [comment]
Can't answer for all the types you're asking for, but I can already confirm that rumble (Pokemon Pinball, Perfect Dark, Top Gear Rally), GB Camera, and gyro (Yoshi's Universal Gravitation) were dumped fine here.

I'm curious about Game Boy Advance Video and e-reader too :)

**Comment by PW5190** [comment]
I'm a bit late to the party, but I recently got my Open Source Cartridge Reader. I was able to dump the ROM for Kirby Tilt 'n' Tumble, but not the save file. Heck, I wasn't even able to dump the ROM for the GBA Video Shrek + Shark Tale 2-in-1 Movie cart.

```
Cart has no Sram

Press Button...
```
This is the error message I got when I attempted to back up my Kirby Tilt 'n' Tumble save file.

Any chance this issue can be fixed? I really need to backup my save file. As far to my knowledge, this is one of the few Game Boy (Color) games to not use battery-backed SRAM for saving, and instead utilizing EEPROM. And this is one of two Game Boy Color games to have a tilt sensor. The other game that has it is Enix's Command Master.
